pkg/pool-weighted/test/WeightedPool.test.ts

```diff
@@ -42,17 +42,22 @@ describe('WeightedPool', function () {
 
     const initialBalances = range(1, 3).map(fp);
 
-    sharedBeforeEach('deploy pool', async () => {
+    sharedBeforeEach('deploy pool and real vault', async () => {
+      const vault = await Vault.create();
+
       tokens = allTokens.subset(2);
+      await tokens.mint({ to: lp.address, amount: max(initialBalances) });
+      await tokens.approve({ to: vault.address, from: lp, amount: max(initialBalances) });
```

Comment:
Is this so that tests don't revert when calling `manageUserBalance`?

Reply:
I would add it to the mock vault in master. For the patch, this should do just to turn the CI green, since only this test seems to be affected.